### PR TITLE
Update ca_ES to CLDR Version 32

### PR DESCRIPTION
**Pull request by rbuj**
http://www.unicode.org/cldr/charts/latest/summary/ca.html

### CODE REVIEW

File: localedata/locales/ca_ES [hunk 1]
```diff
@@ -114,12 +114,18 @@ day     "diumenge";/
         "dijous";/
         "divendres";/
         "dissabte"
-abmon   "gen";"feb";/
-        "mar";"abr";/
-        "mai";"jun";/
-        "jul";"ago";/
-        "set";"oct";/
-        "nov";"des"
+alt_abmon "gen.";"febr.";/
```
Comment (rluzynski) [inline localedata/locales/ca_ES:117]: This should be ```ab_alt_mon```.

Also I must rethink the number of spaces to indent.

File: localedata/locales/ca_ES [hunk 2]
```diff
@@ -144,7 +150,7 @@ mon     "de gener";/
         "d<U2019>octubre";/
         "de novembre";/
         "de desembre"
-d_t_fmt "%a %d %b %Y %T %Z"
+d_t_fmt "%A, %e %B de %Y, %T %Z"
```
Comment (rluzynski) [inline localedata/locales/ca_ES:153]: Looks good if this is what you want. ```"%d"``` displays a day number with leading zero: ```"01"```, ```"02"```... ```"%e"``` displays a day number with leading space: ```" 1"```, ```" 2"```... If you want no leading zero and no leading space then you should use ```"%-d"``` or ```"%-e"```, both produce the same result.

**Review by rluzynski — CHANGES_REQUESTED** [review]
I don't think it makes sense to make a pull request here because this is just my private fork, that means nobody will use it except me. The updates should go to [SourceWare.org bugzilla](https://sourceware.org/bugzilla/) and [libc-alpha](https://sourceware.org/ml/libc-alpha/) mailing list but I can handle this.

**Comment by rluzynski** [comment]
Weekdays update is OK, maybe those two commits should be squashed.

**Comment by rbuj** [comment]
Ok, thank you very much ;-)

**Pull request closed without merging**